app/views/super_admin/content.py

```diff
@@ -71,15 +72,20 @@ def placeholder_upload():
             wpercent = (basewidth / float(img.size[0]))
             hsize = int((float(img.size[1]) * float(wpercent)))
             img = img.resize((basewidth, hsize), PIL.Image.ANTIALIAS)
-            os.mkdir(app.config['TEMP_UPLOADS_FOLDER'])
+            try:
+                os.mkdir(app.config['TEMP_UPLOADS_FOLDER'])
```

Comment:
I will check the error in my local system once and then approve it. @niranjan94 feel free to approve if you have checked.